tests/acceptance/view-user-profile-test.js

```
@@ -3,7 +3,7 @@ import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
 import profilePage from 'codecrafters-frontend/tests/pages/settings/profile-page';
 import userPage from 'codecrafters-frontend/tests/pages/user-page';
 import { assertTooltipContent } from 'ember-tooltips/test-support';
-import { currentURL } from '@ember/test-helpers';
+import { currentURL, pauseTest } from '@ember/test-helpers';
```

ESLint

[error] 'pauseTest' is defined but never used.

(no-unused-vars)